spec/lib/booker/v4.1/customer_spec.rb

```diff
@@ -272,6 +273,19 @@
     end
   end
 
+  describe '#treatment' do
+    let(:expected_params) { {access_token: 'access_token' } }
+
+    before do
+      expect(client).to receive(:get)
+                          .with('/v4.1/customer/treatment/123', expected_params, Booker::V4::Models::TreatmentVerifiedBookableOnline).and_return(response)
+    end
+
+    it 'returns appointment' do
```

Comment:
I think this should say 'returns treatment'

Reply:
ops haha